### PR TITLE
sc-76288-sc-76287-update-navbar-and-footer-links-to-focus-on-researchers

### PR DESCRIPTION
**Pull request by ashishks0522**
## Stories
- [sc-76287-update-navbar-links-in-sphinx](https://app.shortcut.com/xanaduai/story/76287/update-navbar-links-in-sphinx)
- [sc-76288-update-footer-links-to-include-research-page](https://app.shortcut.com/xanaduai/story/76288/update-footer-links-to-include-research-page-in-sphinx)

## Changes
- Updated navbar to be more researcher focused  - similar to https://github.com/XanaduAI/pennylane.ai-react/pull/1829
- Updated footer links to include `Research` page link  - similar to https://github.com/XanaduAI/pennylane.ai-react/pull/1834
- Removed trailing slash from PennyLane website URLs

Screenshot 2024-11-01 at 1 56 39 PM

Screenshot 2024-11-01 at 3 55 37 PM
Screenshot 2024-11-01 at 3 55 33 PM
Screenshot 2024-11-01 at 3 55 29 PM
Screenshot 2024-11-01 at 3 55 24 PM

### CODE REVIEW

File: .github/CHANGELOG.md
Comment (Mandrenkov) [inline .github/CHANGELOG.md:1]: Don't forget to add yourself as a contributor! 👻

**Review by Mandrenkov — APPROVED** [review]

**Comment by Lorraine-at-Xanadu** [comment]
Based on the screenshots, the Sphinx navbar and footer looks good, @ashishks0522. If you can confirm that 'How-to' links to the 'Results' page with `Demos` and the `How-to` category filter already applied, I can approve this PR (see https://github.com/XanaduAI/pennylane.ai-react/pull/1829#issuecomment-2452544110 for reference).

**Review by Alan-eMartin — APPROVED** [review]
Looks good

**Comment by ashishks0522** [comment]
> Based on the screenshots, the Sphinx navbar and footer looks good, @ashishks0522. If you can confirm that 'How-to' links to the 'Results' page with `Demos` and the `How-to` category filter already applied, I can approve this PR (see [XanaduAI/pennylane.ai-react#1829 (comment)](https://github.com/XanaduAI/pennylane.ai-react/pull/1829#issuecomment-2452544110) for reference).

Just pushed that change, it should link correctly now 🙂

**Comment by Mandrenkov** [comment]
Hey @Lorraine-at-Xanadu and @ashishks0522, as an FYI, there are PR preview links for changes to the PST (and XST too)! They're just a little tricky to find. I've attached a screenshot to help. For this PR, the link is [here](https://xanaduai-pennylane--68.com.readthedocs.build/projects/sphinx-theme/en/68/).

image

**Review by Lorraine-at-Xanadu — CHANGES_REQUESTED** [review]
Hi @ashishks0522! There are a couple of links that don't work, but I'm not sure if it's just in the PR preview.

1. Clicking on `Why PennyLane` should link to the new 'Research' page.
2. Clicking on `Ecosystem` should link to the 'Get involved' page. Currently, it links to a 404 Read the Docs page (see screenshot).

image

File: pennylane_sphinx_theme/navbar.py [hunk 1]
```diff
@@ -4,98 +4,127 @@
...
-                "href": "https://pennylane.ai/plugins/",
+                "name": "GitHub",
+                "href": "https://github.com/PennyLaneAI/pennylane",
+                "external": True,
+            },
+        ],
+    },
+    {
+        "name": "Ecosystem",
+        "href": "/get-involved",
```
[the remark this answers is not in the document]
Comment (ashishks0522) [reply pennylane_sphinx_theme/navbar.py]: Thanks @Mandrenkov, copy paste from `pennylane.ai-react` got me there 😬

**Comment by ashishks0522** [comment]
> Clicking on Why PennyLane should link to the new 'Research' page.

@Lorraine-at-Xanadu this will work once the research page is live, right now Research page changes are in a PR to be merged in. I have tagged you in the final [PR](https://github.com/XanaduAI/pennylane.ai-react/pull/1824) for QA and mentioned these caveats in the description.

Other links should work as expected now, thanks @Mandrenkov for the suggestions.

**Review by Mandrenkov — APPROVED** [review]

**Review by Lorraine-at-Xanadu — APPROVED** [review]
This looks great, @ashishks0522. Thank you very much.

**Pull request merged**